pkg/controller/volume/persistentvolume/controller.go

```diff
@@ -964,7 +964,7 @@ func (ctrl *PersistentVolumeController) recycleVolumeOperation(arg interface{})
 	volume = newVolume
 
 	// Find a plugin.
-	spec := vol.NewSpecFromPersistentVolume(volume, false)
+	spec := vol.NewSpecFromPersistentVolume(volume, false, "")
```

Comment:
same thing in pkg/controller/volume/persistentvolume/controller.go:1123 (doDeleteVolume)